tests/Nox.Reference.Data.Machine.Tests/MacAddresses.Tests.cs

```diff
@@ -27,19 +30,11 @@ public void Setup()
         string expectedPrefix,
         string expectedOrganizationName)
     {
-        var info = _macAddressService.GetMacAddressInfo(input);
+        var info = _macAddressContext.MacAddresses.Get(input);
 
         Assert.That(info, Is.Not.Null);
+        Assert.That(info.Id, Is.EqualTo(expectedPrefix));
         Assert.That(info.MacPrefix, Is.EqualTo(expectedPrefix));
         Assert.That(info.OrganizationName, Is.EqualTo(expectedOrganizationName));
     }
-
-    [TestCase("V0:16:F6:11:22:33")]
-    [TestCase("0016-F6-11-22-33")]
-    [TestCase("")]
-    [TestCase(null)]
-    public void GetVendorMacAddress_InvalidMacAddressString_ShouldThrow(string input)
-    {
-        Assert.Throws<ArgumentException>(() => _macAddressService.GetMacAddressInfo(input));
```

Comment:
hm, should we have a new case for this?

Reply:
We should think how we are going to catch exceptions on db context. Whether we expect to handle them

Reply:
It's easier to uncomment and change peace of code. The conditions will remain the same